cmd/lambda/leases/main.go

```diff
@@ -127,6 +147,52 @@ func init() {
 	muxLambda = gorillamux.New(r)
 }
 
+// initConfig configures package-level variables
+// loaded from env vars.
+func initConfig() {
+	cfgBldr := &config.ConfigurationBuilder{}
+	Settings = &leaseControllerConfiguration{}
+	if err := cfgBldr.Unmarshal(Settings); err != nil {
+		log.Fatalf("Could not load configuration: %s", err.Error())
+	}
+
+	// load up the values into the various settings...
+	err := cfgBldr.WithEnv("AWS_CURRENT_REGION", "AWS_CURRENT_REGION", "us-east-1").Build()
+	if err != nil {
+		log.Printf("Error: %+v", err)
+	}
+	svcBldr := &config.ServiceBuilder{Config: cfgBldr}
+
+	_, err = svcBldr.
+		// AWS services...
+		WithDynamoDB().
+		WithSTS().
+		WithS3().
+		WithSNS().
+		WithSQS().
+		// DCE services...
+		WithStorageService().
```

Comment:
I'm not sure we need this.  I would prefer to add it back as we use it?  But nothing too critical here.

Reply:
fixed